scripts/pylib/twister/twisterlib/reports.py

```diff
@@ -474,24 +477,31 @@ def summary(self, results, unrecognized_sections, duration):
                 Fore.RED if results.error else Fore.RESET,
                 results.error,
                 Fore.RESET,
-                results.skipped_configs,
+                results.notrun,
                 Fore.YELLOW if self.plan.warnings else Fore.RESET,
                 self.plan.warnings,
                 Fore.RESET,
                 duration))
 
         total_platforms = len(self.platforms)
-        # if we are only building, do not report about tests being executed.
-        if self.platforms and not self.env.options.build_only:
-            logger.info("In total {} test cases were executed, {} skipped on {} out of total {} platforms ({:02.2f}%)".format(
-                results.cases - results.skipped_cases,
-                results.skipped_cases,
-                len(self.filtered_platforms),
-                total_platforms,
-                (100 * len(self.filtered_platforms) / len(self.platforms))
-            ))
-
-        built_only = results.total - run - results.skipped_configs
+        # FIXME
```

Comment:
If we want to keep `FIXME` can we at least add a hint about what need to be fixed?